Doc/library/email.parser.rst

```diff
@@ -265,6 +265,9 @@ in the top-level :mod:`email` package namespace.
    ``Parser().parsestr(s)``.  *_class* and *policy* are interpreted as
    with the :class:`~email.parser.Parser` class constructor.
 
+   To decode message headers, set policy as 'email.policy.default'. For
```

Comment:
```suggestion
   To decode message headers, set *policy* as ``email.policy.default``. For
```

Reply:
Perhaps 'set policy *to* ...' would flow a bit better as well. In the issue, Hrvoje also suggested [some improvements](https://github.com/python/cpython/issues/61707#issuecomment-1093611366) to `Parser` that seems useful as well.